core-plugins/src/e2e-test/resources/testdata/file/expected_outputs/OUTPUT_FOR_ROWDENO_TEST.csv

```diff
@@ -0,0 +1,3 @@
+2,San Francisco
+3,Los Angeles
+1,New York
```

Comment:
why does the output file not have column name `newCity`?

Reply:
added the column name .Previously we are skipping the header now it is enabled.